Index: openssl-1.0.2d/crypto/armcap.c
===================================================================
--- openssl-1.0.2d.orig/crypto/armcap.c
+++ openssl-1.0.2d/crypto/armcap.c
@@ -153,10 +153,12 @@ void OPENSSL_cpuid_setup(void)
             OPENSSL_armcap_P |= ARMV8_SHA256;
         }
     }
+/*
     if (sigsetjmp(ill_jmp, 1) == 0) {
         _armv7_tick();
         OPENSSL_armcap_P |= ARMV7_TICK;
     }
+*/
 
     sigaction(SIGILL, &ill_oact, NULL);
     sigprocmask(SIG_SETMASK, &oset, NULL);
