Index: openssl-1.0.2d/crypto/cversion.c
===================================================================
--- openssl-1.0.2d.orig/crypto/cversion.c
+++ openssl-1.0.2d/crypto/cversion.c
@@ -67,15 +67,7 @@ const char *_current_SSLeay_version(int
     if (t == SSLEAY_VERSION)
         return OPENSSL_VERSION_TEXT;
     if (t == SSLEAY_BUILT_ON) {
-#ifdef DATE
-# ifdef OPENSSL_USE_BUILD_DATE
-        return (DATE);
-# else
-        return ("built on: reproducible build, date unspecified");
-# endif
-#else
         return ("built on: date not available");
-#endif
     }
     if (t == SSLEAY_CFLAGS) {
 #ifdef CFLAGS
