Index: openssl-1.0.1f/ssl/ssltest.c
===================================================================
--- openssl-1.0.1f.orig/ssl/ssltest.c
+++ openssl-1.0.1f/ssl/ssltest.c
@@ -1076,19 +1076,8 @@ bad:
 #ifndef OPENSSL_NO_KRB5
 	if (c_ssl  &&  c_ssl->kssl_ctx)
                 {
-                char	localhost[MAXHOSTNAMELEN+2];
-
-		if (gethostname(localhost, sizeof localhost-1) == 0)
-                        {
-			localhost[sizeof localhost-1]='\0';
-			if(strlen(localhost) == sizeof localhost-1)
-				{
-				BIO_printf(bio_err,"localhost name too long\n");
-				goto end;
-				}
 			kssl_ctx_setstring(c_ssl->kssl_ctx, KSSL_SERVER,
-                                localhost);
-			}
+                                "localhost");
 		}
 #endif    /* OPENSSL_NO_KRB5  */
 
