Index: openssl-1.0.1k/crypto/cversion.c
===================================================================
--- openssl-1.0.1k.orig/crypto/cversion.c
+++ openssl-1.0.1k/crypto/cversion.c
@@ -68,11 +68,7 @@ const char *_current_SSLeay_version(int
 		return OPENSSL_VERSION_TEXT;
 	if (t == SSLEAY_BUILT_ON)
 		{
-#ifdef DATE
-		return(DATE);
-#else
 		return("built on: date not available");
-#endif
 		}
 	if (t == SSLEAY_CFLAGS)
 		{
