diff --git a/include/openssl/opensslv.h b/include/openssl/opensslv.h
index 7cf31d3625..a1bdc4bdbf 100644
--- a/include/openssl/opensslv.h
+++ b/include/openssl/opensslv.h
@@ -40,7 +40,7 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x1010108fL
-# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1h  22 Sep 2020"
+# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1h  FIPS 22 Sep 2020"
 
 /*-
  * The macros below are to be used for shared library (.so, .dll, ...)
