diff -up openssl-1.1.1q/include/openssl/opensslv.h.version-override openssl-1.1.1q/include/openssl/opensslv.h
--- openssl-1.1.1q/include/openssl/opensslv.h.version-override	2022-07-07 13:14:40.123541142 +0200
+++ openssl-1.1.1q/include/openssl/opensslv.h	2022-07-07 13:15:20.777288763 +0200
@@ -40,7 +40,7 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x1010117fL
-# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1w  11 Sep 2023"
+# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.1w  FIPS 11 Sep 2023"
 
 /*-
  * The macros below are to be used for shared library (.so, .dll, ...)
